src/components/styledComponents.js

```diff
@@ -0,0 +1,52 @@
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/spacex/pullRequest?prid=7693722)
```suggestion
import styled from "styled-components";
```